modules/nav/NavbarPendingRewards.tsx

```diff
@@ -59,18 +60,14 @@ export function NavbarPendingRewards() {
     const totalPendingRewardsUSD = pendingRewardsTotalUSD + pendingReliquaryRewardsTotalUSD;
 
     const { data: contractCalls } = useGaugeClaimGetContractCallData(
-        totalPendingRewardsUSD > 0.01,
-        pendingBALUSD > 0.01,
+        // totalPendingRewardsUSD > 0.01,
```

Comment:
Remove these if not needed anymore?

Reply:
Good catch. Used that for testing with my shrimp rewards.